lib/widgets/content.dart

```diff
@@ -248,25 +248,245 @@ class MessageImage extends StatelessWidget {
+// .sb { color: hsl(86deg 57% 40%); }
+final _kCodeBlockStyleSb = TextStyle(color: const HSLColor.fromAHSL(1, 86, 0.57, 0.40).toColor());
+// .sc { color: hsl(86deg 57% 40%); }
+final _kCodeBlockStyleSc = TextStyle(color: const HSLColor.fromAHSL(1, 86, 0.57, 0.40).toColor());
+// .sd { color: hsl(86deg 57% 40%); font-style: italic; }
+final _kCodeBlockStyleSd = TextStyle(color: const HSLColor.fromAHSL(1, 86, 0.57, 0.40).toColor(), fontStyle: FontStyle.italic);
+// .s2 { color: hsl(225deg 71% 33%); }
+final _kCodeBlockStyleS2 = TextStyle(color: const HSLColor.fromAHSL(1, 225, 0.71, 0.33).toColor());
+// .se { color: hsl(26deg 69% 43%); font-weight: bold; }
+final _kCodeBlockStyleSe = TextStyle(color: const HSLColor.fromAHSL(1, 26, 0.69, 0.43).toColor(), fontWeight: FontWeight.bold);
+// .sh { color: hsl(86deg 57% 40%); }
+final _kCodeBlockStyleSh = TextStyle(color: const HSLColor.fromAHSL(1, 86, 0.57, 0.40).toColor());
+// .si { color: hsl(336deg 38% 56%); font-weight: bold; }
+final _kCodeBlockStyleSi = TextStyle(color: const HSLColor.fromAHSL(1, 336, 0.38, 0.56).toColor(), fontWeight: FontWeight.bold);
+// .sx { color: hsl(120deg 100% 25%); }
+final _kCodeBlockStyleSx = TextStyle(color: const HSLColor.fromAHSL(1, 120, 1, 0.25).toColor());
+// .sr { color: hsl(189deg 54% 49%); }
+final _kCodeBlockStyleSr = TextStyle(color: const HSLColor.fromAHSL(1, 189, 0.54, 0.49).toColor());
+// .s1 { color: hsl(86deg 57% 40%); }
+final _kCodeBlockStyleS1 = TextStyle(color: const HSLColor.fromAHSL(1, 86, 0.57, 0.40).toColor());
+// .ss { color: hsl(241deg 68% 28%); }
+final _kCodeBlockStyleSs = TextStyle(color: const HSLColor.fromAHSL(1, 241, 0.68, 0.28).toColor());
+// .bp { color: hsl(120deg 100% 25%); }
+final _kCodeBlockStyleBp = TextStyle(color: const HSLColor.fromAHSL(1, 120, 1, 0.25).toColor());
+// .vc { color: hsl(241deg 68% 28%); }
+final _kCodeBlockStyleVc = TextStyle(color: const HSLColor.fromAHSL(1, 241, 0.68, 0.28).toColor());
+// .vg { color: hsl(241deg 68% 28%); }
+final _kCodeBlockStyleVg = TextStyle(color: const HSLColor.fromAHSL(1, 241, 0.68, 0.28).toColor());
+// .vi { color: hsl(241deg 68% 28%); }
+final _kCodeBlockStyleVi = TextStyle(color: const HSLColor.fromAHSL(1, 241, 0.68, 0.28).toColor());
+// .il { color: hsl(0deg 0% 40%); }
+final _kCodeBlockStyleIl = TextStyle(color: const HSLColor.fromAHSL(1, 0, 0, 0.40).toColor());
+
 class CodeBlock extends StatelessWidget {
   const CodeBlock({super.key, required this.node});
 
   final CodeBlockNode node;
 
   @override
   Widget build(BuildContext context) {
-    final text = node.text;
-
     return Container(
-      padding: const EdgeInsets.fromLTRB(7, 5, 7, 3),
       decoration: BoxDecoration(
         color: Colors.white,
         border: Border.all(
           width: 1,
-          color: const HSLColor.fromAHSL(0.15, 0, 0, 0).toColor())),
+          color: const HSLColor.fromAHSL(0.15, 0, 0, 0).toColor()),
+        borderRadius: BorderRadius.circular(4)),
```

Comment:
This is a good change, thanks. (I think this border radius must be one of the changes that happened in the recent web redesign; see #157.)

Let's put it in a separate commit, though, so it can get its own commit message. See the Zulip style guide for commits in a branch:
https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html

Reply:
ack, moved to a [separate commit](https://github.com/zulip/zulip-flutter/pull/242/commits/50c6be84b4a2a41db655c41b2acc1621a6353d18)